packages/transport/scripts/protobuf-build.sh

```diff
@@ -35,7 +35,7 @@ grep -hv -e '^import ' -e '^syntax' -e '^package' -e 'option java_' "$SRC"/messa
 | grep -v '    reserved '>> "$DIST"/messages.proto
 
 # BUILD messages.json from message.proto
-node_modules/.bin/pbjs -t json -p "$DIST" -o "$DIST"/messages.json --keep-case messages.proto
+../../../node_modules/.bin/pbjs -t json -p "$DIST" -o "$DIST"/messages.json --keep-case messages.proto
```

Comment:
`./scripts/protobuf-build.sh: line 38: ../../../node_modules/.bin/pbjs: No such file or directory`

Reply:
it does something with 
```suggestion
../../node_modules/.bin/pbjs -t json -p "$DIST" -o "$DIST"/messages.json --keep-case messages.proto
```

error:
<img width="1285" alt="Screenshot 2022-12-13 at 14 03 42" src="https://user-images.githubusercontent.com/33235762/207325492-58647610-44bb-4248-9d60-c19cdbf64745.png">